### ISSUE TITLE
Database Security

### ISSUE THREAD

**Issue by michaelfitzhavey** [holistic-web/website-rachel-lewis#72]
The firebase database must be locked down so that only authenticated users can read or write data.

**Comment by andrewlew1s** [comment]
Do we have authentication for this site yet?

**Comment by michaelfitzhavey** [comment]
@andrewlew1s nope, but I believe this means someone could theoretically connect to our firebase db with their own app and make changes.
